include/omnidirectional_controllers/omnidirectional_controller.hpp

```diff
@@ -58,13 +60,17 @@ class OmnidirectionalController : public controller_interface::ControllerInterfa
   ~OmnidirectionalController();
 
  protected:
-  struct WheelHandle {
-    std::reference_wrapper<const hardware_interface::LoanedStateInterface> velocity_state;
-    std::reference_wrapper<hardware_interface::LoanedCommandInterface> velocity_command;
-  };
 
   std::vector<std::string> wheel_names_;
-  std::vector<WheelHandle> registered_wheel_handles_;
+  std::vector<StateInterfaceReferenceWrapper> registered_wheel_state_ifs_;
+  std::vector<CommandInterfaceReferenceWrapper> registered_wheel_cmd_ifs_;
+  // struct WheelHandle {
+  //   std::reference_wrapper<const hardware_interface::LoanedStateInterface> velocity_state;
+  //   std::reference_wrapper<hardware_interface::LoanedCommandInterface> velocity_command;
+  // };
+
+  // std::vector<std::string> wheel_names_;
+  // std::vector<WheelHandle> registered_wheel_handles_;
```

Comment:
Could you, please, remove the commented lines?

Reply:
It is corrected